src/CompanyService.Data/Interfaces/IDepartmentUserRepository.cs

```diff
@@ -0,0 +1,14 @@
+﻿using LT.DigitalOffice.CompanyService.Models.Db;
+using LT.DigitalOffice.Kernel.Attributes;
+using System;
+
+namespace LT.DigitalOffice.CompanyService.Data.Interfaces
+{
+    [AutoInject]
+    public interface IDepartmentUserRepository
+    {
+        public bool Add(DbDepartmentUser departmentUser);
```

Comment:
remove public and add empty separated lines

Reply:
fixed